package.json

```
@@ -1,13 +1,48 @@
 {
   "name": "@dapr/testcontainer-node",
-  "version": "1.0.0",
-  "description": "Dapr TestContainer for Node JS",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "0.5.0",
+  "description": "Dapr TestContainer for NodeJS",
   "keywords": [],
-  "author": "Dapr Authors",
   "license": "Apache-2.0",
-  "packageManager": "pnpm@10.14.0"
+  "author": "Dapr Authors",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dapr/testcontainer-node.git"
+  },
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "pretty": "prettier --list-different \"**/*.{ts,js,json,md}\"",
```

```suggestion
    "prepack": "npm run build",
    "pretty": "prettier --list-different \"**/*.{ts,js,json,md}\"",
```

@WhitWaldo Add this to make sure build is run before publish